markdown/3.x/en/documentation/tracking-guide.md

```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send tracking request to the different channel, use `channel` attribute
+```html
+<ff-checkout-tracking disable-auto-tracking>
+   <ff-checkout-tracking-item record-id="10321926" count="5" channel="new-channel-name"></ff-checkout-tracking-item>
+</ff-checkout-tracking>
+```
+If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```

Comment:
[OPITONAL]
I think I have already replaced `poke around` with some more official expression. Please, check if there are more occurrences in the documentation and replace if applicable. A good chance to improve it a little.